src/Export/Manager.php

```diff
@@ -130,31 +131,83 @@ public function exportLocalEntity(
       );
     }
 
+    // Build the field maping for this local entity.
     $field_mapping = $this->fieldMapping($local_entity, $sync);
-
-    if(!$field_mapping) {
+    if (!$field_mapping) {
       return;
     }
-    $remote_entity = constructEntityObject($local_entity, $field_mapping);
 
+    // Now, construct an object with the values to pass to the remote.
+    $remote_entity = $this->constructEntityObject(
+      $local_entity,
+      $field_mapping,
+      $sync
+    );
 
-    // Now, use the remote client to fetch the remote entity for this ID.
-    $remote_entity = $this->clientFactory
+    // Now, use the client to export out the entity with values to the remote.
+    $response = $this->clientFactory
       ->getByClientConfig($entity_mapping['client'])
-      ->exportEntity($remote_entity, $entity_mapping['entity_id']);
+      ->exportEntity($remote_entity, $entity_mapping['id']);
 
+
+    // @TODO: Finally, update the local entity, ie. save the remote ID, etc.
```

Comment:
@barnettech I've added these last TODOs, which I'm hoping you can look into adding.

The first TODO, this is where, when we export the local entity out to the API and get a response, we would most likely will also be getting the just saved entity back from the remote. So, if it's a new entity that we just created, we would get the entity from the remote back and need to save the remote ID back to our local entity. So, a function to do that would need to be added here. See `setRemoteIdField()` and `setRemoteChangedField()` functions in the Import manager.

Second, we need to make a terminate event call like we're doing at the end of the importLocalEntity() function in the Import manager.